wintercms/library
# Namespace alias support in extendable trait

This PR adds support for `ExtendableTrait::isClassExtendedWith` to use namespace aliases registered within the `ClassLoader`. This fixes the issue @mjauvin found with `Winter.Translate` not working correctly with RainLab implements.

To test (provided by @mjauvin):

```
1. install rainlab.user, rainlab.location, rainlab.translate, offline.mall
2. run the migrations
3. in the backend, Settings->Translate->Manage languages
4. add a second language and enable it
5. go to the Catalogue menu and click `Create` button to create a new Product.
6. notice the language selector in the Product name field (among others).
```
To trigger the issue:
```
1. install winter.translate
2. run migrations
3. go back to Catalogue
4. click `Create` button
5. notice the Product name field is missing the language selector
```

The issue was caused by this code:

![image](https://user-images.githubusercontent.com/31214002/116328973-20293e80-a7c2-11eb-8867-62824f1911aa.png)

Within: `plugins/winter/translate/classes/EventRegistry.php`
